index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
```

Comment:
I think you did a great job however there is still missing one thing according to the recommendations given by Microverse. Kindly add why you suggest this change, It might be obvious for you but not for someone else.
Please do this in all your comments where you suggest the use of classes instead of IDs. :+1: :smiley: :smiley: